### ISSUE TITLE
Library Authoring UAT Feedback: Links must appear in the browser's status bar.

### ISSUE THREAD

**Issue by connorhaugh** [openedx/frontend-app-library-authoring#414]
Hovering over many links (e.g. in the "Libraries" list) do not show the URL that they link to in the browser's status bar. This is an a11y issue - they aren't "real" links; but they should be as they go to a different page.

**Comment by imasdekar** [comment]
@connorhaugh I have added a PR https://github.com/openedx/frontend-app-library-authoring/pull/437 for this issue
